tests/fsharp/tools/eval/test.fsx

```diff
@@ -2285,7 +2285,7 @@ module Query =
             F ([srcTy],[src])
 
     let MakeQueryableTake = 
-        let F = MakeGenericStaticMethod <@ System.Linq.Queryable.Take @> 
+        let F = MakeGenericStaticMethod <@ System.Linq.Queryable.Take : _ * int -> _  @> 
```

Comment:
Why is this necessary now?

Reply:
There are two overloads now, this and the one accepting range, specifying type so it can be properly resolved.